go/cmd/vttestserver/vttestserver_test.go

```diff
@@ -62,6 +65,47 @@ func TestRunsVschemaMigrations(t *testing.T) {
 	assertColumnVindex(t, cluster, columnVindex{keyspace: "test_keyspace", table: "test_table1", vindex: "my_vdx", vindexType: "hash", column: "id"})
 }
 
+func TestCanVtGateExecute(t *testing.T) {
+	cluster, err := startCluster()
+	assert.NoError(t, err)
+	defer cluster.TearDown()
+	args := os.Args
+	defer resetFlags(args)
+
+	client, err := vtctlclient.New(fmt.Sprintf("localhost:%v", cluster.GrpcPort()))
+	assert.NoError(t, err)
+	defer client.Close()
+	stream, err := client.ExecuteVtctlCommand(
+		context.Background(),
+		[]string{
+			"VtGateExecute",
+			"-server",
+			fmt.Sprintf("localhost:%v", cluster.GrpcPort()),
+			"select 'success';",
+		},
+		30*time.Second,
+	)
+	assert.NoError(t, err)
+
+	var b strings.Builder
+	b.Grow(1024)
+
+OUT:
```

Comment:
A tiny nit: the linter doesn't care but I believe it is common to name labels like variables. So `out` or `Out`.